nsds_lab_to_nwb/metadata/resources/list_of_stimuli.yaml

```diff
@@ -41,8 +41,22 @@ wn2:
     parameter_path: ''
 
 dmr:
-    alt_names: []
+    alt_names: ['Ripple']
     description: ''
     audio_path: 'DMR/dmr-500flo-40000fhi-4SM-40TM-40db-96khz-48DF-15min.wav'
     marker_path: 'DMR/trig-dmr-500flo-40000fhi-4SM-40TM-40db-96khz-48DF-15min.wav'
     parameter_path: ''
+
+tone_diagnostic:
+    alt_names: ['Tone diagnostic']
+    description: '' #Vanessa can you please add a description, copy files to catscan and add paths (if the files exist)
```

Comment:
@vngutierrez Can you please add a description, copy files to catscan and add paths (if the files exist)

Reply:
Sure, but I looked and no Tone.mat file exists as far as I know and looked for.

Reply:
@vngutierrez Do you know how the tone diagnostic maps are generated using Max's script? It must know about the stimuli parameters and there probably is a tone diagnostics txt or mat file in his code base.

Reply:
@jthermiz From his matlab script, it shows that there is a `Tone_diag_stim.stimVls.mat`. I would have to look for that on his computer next time I'm on-site.